src/Services/QueuedJobService.php

```diff
@@ -519,6 +519,25 @@ public function checkJobHealth($queue = null)
             ]
         );
 
+        // Send broken job email notification
+        $subject = _t(__CLASS__ . '.BROKEN_JOB', 'Broken job');
+        $message = sprintf(_t(
```

Comment:
Using a combination of sprintf() with _t() is pretty standard elsewhere in the codebase.

You can used named variables with _t(), though that's much less common